cli/lib/kontena_cli.rb

```diff
@@ -1,3 +1,12 @@
+begin
+  unless Gem.loaded_specs.include?('kontena-cli')
+    load File.expand_path('../../kontena-cli.gemspec', __FILE__)
+    KontenaCliGemspec.activate
+    KontenaCliGemspec.activate_dependencies
+  end
+rescue LoadError
+end
+
```

Comment:
Probably missed some "you have unstaged changes and have to stash them" errors when checking out master.